man/sd-journal.xml

```diff
@@ -86,6 +86,16 @@
     — are fully thread-safe and may be called from multiple threads in parallel.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Optional dependencies</title>
+
+    <para>Depending on which build-time options are enabled, functions that operate on
+    <structname>sd_journal</structname> objects might cause optional dependencies to be dynamically loaded
+    via <citerefentry project='die-net'><refentrytitle>dlopen</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    such as decompression libraries (xz, lz4, zstd) or cryptographic primitives libraries (gcrypt).
```

Comment:
hmm, this seems a bit random. we load more stuff, no?

Reply:
"cryptographic primitives libraries" is strange. Just "cryptographic libraries" would be better.

Reply:
> hmm, this seems a bit random. we load more stuff, no?

not in libsystemd in the sd_journal APIs, I don't think? This is the sd-journal manpage, and the purpose is to describe what it does when it is used, it's not about the entire project and all other internal binaries